Relatório execução
IDS 
	MAP1
		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map3.map 0 0 255 255
		<255,255,495.0>
		Elapsed time: 177.08348941802979

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map1.map 247 245 191 97
		<191,97,256.0>
		Elapsed time: 28.442903995513916

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map1.map 190 249 175 249
		<175,249,22.0>
		Elapsed time: 0.24417400360107422

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map1.map 235 12 0 154
		failure
		Elapsed time: 141.42143511772156  
	MAP2
		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map2.map 52 97 247 25
		<247,25,306.5>
		Elapsed time: 212.60486006736755 

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map2.map 201 207 84 166
		<84,166,159.5>
		Elapsed time: 16.48185658454895

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map2.map 239 8 6 254                                            failure (Estado inicial ou estado final é bloqueado)
		Elapsed time: 0.01481856584548

	MAP3
		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map3.map 14 4 220 168
		<220,168,488.0>
		Elapsed time: 227.53482818603516

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map3.map 135 6 249 238
		<249,238,361.5>
		Elapsed time: 178.7276611328125

		C:\Users\romeu\Documents\repos\AI\tp1\src>python IDS.py map3.map 32 50 239 131
		failure                                                                                                               Elapsed time: 181.9469699859619 














